### ISSUE TITLE
can't install on osx

### ISSUE THREAD

**Issue by jbsiddall** [dundalek/closh#46]
When attempting to install closh i get the following error:

```
[2018-01-31T22_49_10_691Z-debug.log](https://github.com/dundalek/closh/files/1683596/2018-01-31T22_49_10_691Z-debug.log)
$ npm install -g lumo-cljs closh
/usr/local/bin/closh -> /usr/local/lib/node_modules/closh/bin/closh.js
/usr/local/bin/lumo -> /usr/local/lib/node_modules/lumo-cljs/bin/lumo.js

> lumo-cljs@1.8.0-beta install /usr/local/lib/node_modules/lumo-cljs
> node scripts/npm_install.js || nodejs scripts/npm_install.js


Download failed.
sh: nodejs: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! lumo-cljs@1.8.0-beta install: `node scripts/npm_install.js || nodejs scripts/npm_install.js`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the lumo-cljs@1.8.0-beta install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jsiddall/.npm/_logs/2018-01-31T22_49_10_691Z-debug.log
```

OS: MacOS Sierra 10.12.6
Node: v9.4.0
NPM: 5.6.0

Node & NPM installed via pkg file from https://nodejs.org/en/
I've tried reinstalling Node/NPM multiple times and get the same problem.

**Comment by mnewt** [comment]
This looks like a problem running the lumo `npm_install.js` script. Can you post the debug log somewhere? Not sure if it has information in it that you may consider sensitive.

**Comment by Quezion** [comment]
I found that I couldn't install Closh unless I ensured I had Node version 9.

```
> node --version
v9.11.2
```

It's a slight pain to get Node v9 -- I downloaded node-v9.11.2-darwin-x64.tar.gz  off of https://nodejs.org/download/release/v9.11.2/, unzipped it, and manually symlinked in the /bin/node executable.

(on OSX High Sierra v10.13.3)

**Comment by dundalek** [comment]
These installation issues should be now fixed in the latest release (v0.2.2).
